src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -88,6 +95,7 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
   private expression: string = '';
   private vis: Vis;
   private domNode: any;
+  private labelNode: any;
```

Comment:
```suggestion
  private labelNode?: HTMLDivElement;
```

Reply:
The reason why I used `any` (just like `domNode`) is because we use both `createElement` which creates a `HTMLDivElement` and `ReactDOM.render` which requires `React.DOMElement`. I wasn't sure how to handle this correctly and didn't want to refactor how things work too much.


What do you think?

Reply:
hm, didn't get it:
> ReactDOM.render which requires React.DOMElement

afaik `ReactDOM.render` expect `Element` which is in prototype chain for `HTMLDivElement` 

maybe it was an issue with possible undefined? 
I think this should work: 
```
private labelNode: HTMLDivElement
ReactDOM.render(<EmbeddableErrorLabel />, this.labelNode!); // or explicit if (this.labelNode){ render() } 
```